package.json

```diff
@@ -39,7 +39,7 @@
     "clean": "yarn workspaces run clean",
     "lint": "yarn workspaces run lint",
     "lint:fix": "yarn workspaces run lint:fix",
-    "changeset:version:custom": "yarn changeset version && yarn workspace @sphinx-labs/contracts run write-version"
+    "changeset:version:custom": "yarn changeset version && yarn workspace @sphinx-labs/contracts run write-version && yarn workspace @sphinx-labs/plugins run write-version"
```

Comment:
This allows us to access the current version of the plugins package which we include in the request to the proposal API endpoint. 

Originally, I was going to use this to determine when to deprecate some things in the mono repo related to this change. However, that ended up being unnecessary. I still think it's helpful to know what plugins package versions people are using though, so I decided to keep all of this logic.